lib/ansible/modules/cloud/vmware/vmware_portgroup.py

```diff
@@ -20,70 +21,78 @@
 module: vmware_portgroup
 short_description: Create a VMware portgroup
 description:
-    - Create a VMware portgroup on given host/s or hosts of given cluster
+    - Create a VMware Port Group on a VMware Standard Switch (vSS) for given ESXi host(s) or hosts of given cluster.
 version_added: 2.0
 author:
 - Joseph Callen (@jcpowermac)
 - Russell Teague (@mtnbikenc)
-- Abhijeet Kasurde (@Akasurde) <akasurde@redhat.com>
+- Abhijeet Kasurde (@Akasurde)
+- Christian Kotte (@ckotte)
 notes:
-    - Tested on vSphere 5.5, 6.5
+    - Tested on vSphere 5.5 and 6.5
+    - Complete configuration only tested on vSphere 6.5
 requirements:
     - "python >= 2.6"
     - PyVmomi
 options:
-    switch_name:
+    switch:
         description:
             - vSwitch to modify.
         required: True
-    portgroup_name:
+        aliases: [ 'switch_name', 'vswitch' ]
+    portgroup:
         description:
             - Portgroup name to add.
         required: True
+        aliases: [ 'portgroup_name' ]
     vlan_id:
         description:
             - VLAN ID to assign to portgroup.
-        required: True
-    network_policy:
+            - Set to 0 (no VLAN tagging) by default.
+        required: False
+        default: 0
+        aliases: [ 'vlan' ]
+    security:
         description:
             - Network policy specifies layer 2 security settings for a
               portgroup such as promiscuous mode, where guest adapter listens
               to all the packets, MAC address changes and forged transmits.
             - Dict which configures the different security values for portgroup.
             - 'Valid attributes are:'
-            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: false)'
-            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: false)'
-            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: false)'
+            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: None)'
+            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: None)'
+            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: None)'
         required: False
-        version_added: "2.2"
-        default: {
-            mac_changes: false,
-            promiscuous_mode: false,
-            forged_transmits: false,
-        }
-    teaming_policy:
+        version_added: "2.8"
+        aliases: [ 'security_policy' ]
```

Comment:
Add `network_policy` as well so that we keep backward compatibility.

Reply:
done